### ISSUE TITLE
Can use come custom tag (class created by me). LIKE: export const basicComponentHtml = { hoverComponent: '', trigger: 'hover me' }

### ISSUE THREAD

**Issue by priyankadevgan1992** [cht8687/react-hover#9]
export const basicComponentHtml = {
  hoverComponent: '<MyOwnClass id='1'  />',
  trigger: 'hover me'
}

**Comment by cht8687** [comment]
@priyankadevgan1992  are you referring to custom classes within the hoverComponent?
  hoverComponent: '<h1> pop up header </h1> <p> pop up content </p>',

could you supply an example? thanks.

**Comment by priyankadevgan1992** [comment]
sure like, I have my own component of my own named as MyComponent,

Below is the code


```
let id='10';   // it will dynamic in my code.
let content = 'Name';

myarr.push(<ReactHover
			className='basic'
			styles={styles.basic}
			componentHtml={{
				hoverComponent: (<MyComponent id={id}> </MyComponent>),
						trigger: content
					}}
					options={optionsCursorTrueWithMargin}
				/>);

<td> {myarr} </td>
```

**Comment by cht8687** [comment]
oh, this could be #3 ,  I am working on that branch [V2.0] (https://github.com/cht8687/react-hover/tree/feature/version2.0)... Still working in progress.The API will be changed in V2 as well to enable render custom component. If you have time and comes with a better solution, you can help to raise a PR!! :smile:

**Comment by priyankadevgan1992** [comment]
oh, sure and thanks for the update.

**Comment by cht8687** [comment]
@priyankadevgan1992 

Hi, I just released the V1.0.0 which support custom components. Please try it and give me some feedback if you can. Cheers.

**Comment by priyankadevgan1992** [comment]
Sure, I will check it out and will let you know if face any other issues
